drivers/mmcsd/mmcsd_sdio.c

```diff
@@ -1251,7 +1251,15 @@ static int mmcsd_transferready(FAR struct mmcsd_state_s *priv)
 
       /* Do not hog the CPU */
 
+#if CONFIG_USEC_PER_TICK <= 1000
       nxsig_usleep(1000);
+#else
+#ifndef CONFIG_MMCSD_CHECK_READY_STATUS_WITHOUT_SLEEP
```

Comment:
```
#ifdef CONFIG_MMCSD_CHECK_READY_STATUS_WITHOUT_SLEEP
      /* Use sched_yield when tick is big to avoid write low speed */

      sched_yield();
#else
     nxsig_usleep(1000);
#endif
```